view/frontend/templates/cards.phtml

```diff
@@ -0,0 +1,57 @@
+<script type="text/javascript">// <![CDATA[
+require([
+        'jquery',
+        'Magento_Ui/js/modal/confirm'
+    ],
+    function($, confirmation) {
+         $('.omise-cc-delete').on('click', function(event){
+             event.preventDefault();
+                 confirmation({
```

Comment:
Indentation seems wrong/weird

Reply:
looks like, I used right click/Format Document in VC. @jonrandy what happens when you right click and format?

Reply:
By hand generally. Rarely if ever use autoformatters